vignettes/programming_strategy.Rmd

```diff
@@ -18,259 +21,328 @@ knitr::opts_chunk$set(
+| `by_vars`          | Variables to group by.                                                                                                                                        |
+| `order`            | List of expressions for sorting a dataset, e.g., `exprs(PARAMCD, AVISITN, desc(AVAL))`.                                                                       |
+| `new_var`          | Name of a single variable to be added to the dataset.                                                                                                         |
+| `new_vars`         | List of variables to be added to the dataset.                                                                                                                 |
+| `new_var_unit`     | Name of the unit variable to be added. It should be the unit of the variable specified for the `new_var` argument.                                            |
+| `filter`           | Expression to filter a dataset, e.g., `PARAMCD == "TEMP"`.                                                                                                    |
+| `start_date`       | The start date of an event/interval. Expects a date object.                                                                                                   |
+| `end_date`         | The end date of an event/interval. Expects a date object.                                                                                                     |
+| `start_dtc`        | (Partial) start date/datetime in ISO 8601 format.                                                                                                             |
+| `dtc`              | (Partial) date/datetime in ISO 8601 format.                                                                                                                   |
+| `date`             | Date of an event / interval. Expects a date object.                                                                                                           |
+| `set_values_to`    | List of variable name-value pairs. Use `process_set_values_to()` for processing the value and providing user friendly error messages.                         |
+| `subject_keys`     | Variables to uniquely identify a subject, defaults to `exprs(STUDYID, USUBJID)`. In function formals, use `subject_keys = get_admiral_option("subject_keys")` |
+| `keep_source_vars` | Specifies which variables from the selected observations should be kept. The default of the argument should be `everything()`.                                |
+| `missing_value`    | A value to be input if the data is missing.                                                                                                                   |
+| `missing_values`   | A named list of expressions where the names are variables in the dataset and the values are a value to be input if the data is missing.                       |
```

Comment:
This is primarily what was changed right? The markdown wrap 72 thing added up top made this a bit difficult to review.

Perhaps for `missing_values` place a small example like below too
 `exprs(BASEC = "MISSING", BASE = -1)`

Reply:
Yep - sounds good, will add.